main.py

```diff
@@ -2,17 +2,14 @@
 An OpenLegend RPG dice bot for Discord servers.
     Copyright (C) 2021  Utku Erol
     Contact: utku.erol@icloud.com
-
     This program is free software: you can redistribute it and/or modify
     it under the terms of the GNU General Public License as published by
     the Free Software Foundation, either version 3 of the License, or
     (at your option) any later version.
-
     This program is distributed in the hope that it will be useful,
     but WITHOUT ANY WARRANTY; without even the implied warranty of
     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
     GNU General Public License for more details.
-
     You should have received a copy of the GNU General Public License
     along with this program.  If not, see <https://www.gnu.org/licenses/>.
 """ 
```

Comment:
I accidentally removed this information in a previous edit, and pasted it back in prior to submitting a pull request.